x-pack/plugins/secops/public/components/page/hosts/uncommon_process_table/index.tsx

```diff
@@ -190,42 +190,45 @@ const getUncommonColumns = (startDate: number): Array<Columns<UncommonProcessesE
     hideForMobile: false,
     render: ({ node }) => {
       const hosts: HostEcsFields[] = node.host;
-      const draggables = hosts.map(({ id, name }) => {
-        if (name != null && id != null) {
+      const draggables = hosts
+        .filter(({ id, name }) => id != null && name != null)
+        .map(({ id, name }, index) => {
           return (
-            <DraggableWrapper
-              key={id}
-              dataProvider={{
-                and: [],
-                enabled: true,
-                id: escapeDataProviderId(`uncommon-process-table-${node._id}-hostName-${name}`),
-                name,
-                excluded: false,
-                kqlQuery: '',
-                queryMatch: {
-                  displayField: 'host.name',
-                  displayValue: name,
-                  field: 'host.id',
-                  value: id,
-                },
-                queryDate: {
-                  from: startDate,
-                  to: Date.now(),
-                },
-              }}
-              render={(dataProvider, _, snapshot) =>
-                snapshot.isDragging ? (
-                  <DragEffects>
-                    <Provider dataProvider={dataProvider} />
-                  </DragEffects>
-                ) : (
-                  <EuiLink href={`#/link-to/hosts/${encodeURIComponent(id)}`}>{name}</EuiLink>
-                )
-              }
-            />
+            <React.Fragment key={index}>
+              {index !== 0 ? <>,&nbsp;</> : null}
```

Comment:
So it looks like you're supposed to be able to accomplish this with just a [reduce](https://stackoverflow.com/a/47459785), but in my testing it would always remove the trailing space without additional wrappers/tricks that make it even more cumbersome. This seems cleanest for dealing with that -- nice! 👍